dev/bots/firebase_testlab.sh

```diff
@@ -30,23 +30,29 @@ function test_app_bundle() {
   pushd "$@"
   ../../../bin/flutter build appbundle --target-platform android-arm,android-arm64
 
+  aab="build/app/outputs/bundle/release/app-release.aab"
+
+  # If the app bundle doesn't exist, then exit with code 1.
+  if [ ! -f "$aab" ]; then
+    exit 1
+  fi
   # Firebase Test Lab tests are currently known to be failing with
   # "Firebase Test Lab infrastructure failure: Error during preprocessing"
   # Remove "|| exit 0" once the failures are resolved
   # https://github.com/flutter/flutter/issues/36501
 
   # Run the test.
   gcloud firebase test android run --type robo \
-    --app build/app/outputs/bundle/release/app.aab \
+    --app "$aab" \
     --timeout 2m \
     --results-bucket=gs://flutter_firebase_testlab \
     --results-dir="$@"/"$GIT_REVISION"/"$CIRRUS_BUILD_ID" || exit 0
 
```

Comment:
Should we get rid of the exit 0 here?

Reply:
Yep. I did remove it. I also added a device flag.